cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js

```diff
@@ -49,5 +55,15 @@ describe('bulk-edit', () => {
       BulkEditActions.verifyCheckedDropdownMenuItem();
       BulkEditActions.verifyUncheckedDropdownMenuItem();
     });
+
+    it('C360556 Populating preview of matched records in case no matches (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
+      BulkEditSearchPane.selectRecordIdentifier('User Barcodes');
+
+      BulkEditSearchPane.uploadFile(invalidUserBarcodesFileName);
+      BulkEditSearchPane.waitFileUploading();
+
```

Comment:
I don't see here a check that there is no records fould from the step1: No records are matched and the label reads: 0 records match

Reply:
fixed